test/viewModels/regulatorsAndDocuments/CharityInformationSummaryHelperSpec.scala

```diff
@@ -104,5 +105,21 @@ class CharityInformationSummaryHelperSpec extends SpecBase with SummaryListRowHe
         )
       }
     }
+
+    "For the Can we send letters to this address answer" must {
+
+      "have a correctly formatted summary list row" in {
+
+        helper.canWeSendToThisAddressRow mustBe Seq(
+          summaryListRow(
+           messages("canWeSendLettersToThisAddress.checkYourAnswersLabel"),
+          //  s"test<div>${"123423"}</div>",
```

Comment:
please remove commented line

Reply:
Removed commented line and modified test to reach 100% coverage for this file